sourceinfo/extra_comments_test.txt

```diff
@@ -0,0 +1,725 @@
+desc_test_comments.proto:
```

Comment:
Can this be moved to a testdata directory?

Reply:
Good call. Done